larray/tests/test_array.py

```diff
@@ -2230,9 +2230,9 @@ def test_binary_ops_no_name_axes(self):
         assert_array_equal(raw2_ge_la, raw2 >= raw)
 
     def test_broadcasting_no_name(self):
-        a = ndrange((2, 3))
-        b = ndrange(3)
-        c = ndrange(2)
+        a = ndtest((2, 3))
```

Comment:
This change is wrong. The test was about anonymous axes, which they are no longer.